tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -243,4 +261,16 @@ private[scalanative] object LLVM {
       case Some(tt) => Seq("-target", tt)
       case None     => Seq("-Wno-override-module")
     }
+
+  private def buildCompileOpts(config: Config): Seq[String] =
+    config.compilerConfig.buildTarget match {
+      case BuildTarget.Application    => Seq()
+      case BuildTarget.LibraryDynamic => Seq("-fPIC")
```

Comment:
In the Clang command line docs this shows as `-fpic`. This is to avoid recompiling pre-compiled headers is I am reading this correctly?

Reply:
I based it on this [thread](https://stackoverflow.com/questions/3544035/what-is-the-difference-between-fpic-and-fpic-gcc-parameters). So basically `-fPIC` is safer to use, altought maybe we should try to use `-fpic` when possible.

Reply:
Wow, somehow when I searched this the first time I didn't find `-fPIC` and only found `-fpic`. These are not the same things at all I would guess. The first one have no explanation.
https://clang.llvm.org/docs/ClangCommandLineReference.html